### DISCUSSION TITLE
Fine-tune GLiNER on mostly label empty data

### DISCUSSION THREAD

**Discussion by Lopez-Antonio** [urchade/GLiNER#189] [Q&A]
Hi! I am fine-tuning GLiNER on a dataset where most texts do not present any entity. The model struggles to learn and after a couple of epochs, the f1 score drops to zero. How does GLiNER behave in a scenario like this? Should I keep only the data point where there are entities?

**Comment by urchade** [comment]
Yes, only keep data with entities

**Reply by Lopez-Antonio** [reply]
Thank you for the answer!

I tried this approach, and the f1 does not drop to zero but it is still poor (about 30% while a standard BERT reaches about 60%). The recall is high while the precision is low (80% against 20%). I suppose that since the model is fine-tuned only on data with entities it is loose while making predictions. I will try to still have some empty data. By any chance, do you know some examples where GLiNER was fine-tuned on texts that are mostly without labels?
